### PR TITLE
feat(replays): Improve ReplayTimeline tooltips

### PR DESCRIPTION
**Pull request by ryan953**
We've got:
- Bigger surface areas for the mouse cursor to hover (2px to 4px, woah!)
- Tooltip shows some info about the event (navigation destination, click selector, error value)
- Icons that represent a group of events inherit the color of the 'most interesting' event inside -> we will see more red and green icons, fewer purple.

Screen Shot 2022-05-19 at 10 45 45 PM

Screen Shot 2022-05-19 at 10 45 50 PM

### CODE REVIEW

**Comment by benvinegar** [comment]
This looks great.

**Comment by Jesse-Box** [comment]
I've got some minor UI gripes, but lets get this out the door and focus on the details in a seperate PR.

Would be good to get that X offset sorted before merging, though.

**Comment by ryan953** [comment]
> Would be good to get that X offset sorted before merging, though.

idk what i did, but it's fixed. Blew up the caches, latest code from master (there were some popper related changes, i didn't read though) and it's better now.

**Review by billyvg — APPROVED** [review]

File: static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx [hunk 1]
```diff
@@ -57,31 +56,76 @@ const EventColumn = styled(Timeline.Col)<{column: number}>`
...
+  border-bottom: 1px solid ${p => p.theme.innerBorder};
+  &:last-child {
+    border-bottom: none;
+  }
+`;
+
 const IconPosition = styled('div')`
   position: absolute;
   transform: translate(-50%);
+  margin-left: 2px; /* Half of EVENT_STICK_MARKER_WIDTH */
```
[the remark this answers is not in the document]
Comment (ryan953) [reply static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx]: derp, yeah

**Pull request merged**